packages/node/src/waku/waku-broadcaster-waku-core.ts

```diff
@@ -81,21 +85,20 @@ export class WakuBroadcasterWakuCore {
 
       BroadcasterDebug.log(`Creating waku broadcast client`);
 
-      const peers: string[] = [
+      const bootstrapPeers: string[] = [
         ...WAKU_RAILGUN_DEFAULT_PEERS_NODE,
-        ...WAKU_RAILGUN_DEFAULT_PEERS_WEB,
         ...this.additionalDirectPeers,
       ];
-      const waitTimeoutBeforeBootstrap = 250; // 250 ms - default is 1000ms
+      const waitTimeoutBeforeBootstrap = 1250; // 1250 ms - default is 1000ms
       const waku: LightNode = await createLightNode({
```

Comment:
createLightNode has a bootstrap value that can be used instead of manual libp2p manipulation, unless the custom `waitTimeoutBeforeBootstrap` was shown to be necessary in testing

Reply:
This is done due to the tcp configuration needed by the native modules